run-tests.sh

```diff
@@ -99,13 +99,18 @@ check_isort () {
     isort -rc -c -df --profile black -- **/*.py
 }
 
+check_licenses () {
+    python3 scripts/check_licenses.py
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏After `chmod`-ding the check license script, see the other comment, you can remove reading `python3` here.